webapp/client/src/pages/KrankheitsKostenInfoPage.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import styled from "styled-components";
+import { t } from "i18next";
+import InfoBox from "../components/InfoBox";
+// eslint-disable-next-line import/named
+import { anchorBack, anchorList } from "../lib/contentPagesAnchors";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+
+const ContentWrapper = styled.div`
+  padding-left: var(--spacing-03);
+  padding-right: var(--spacing-03);
+  margin: 0 auto;
+  max-width: var(--main-max-width);
+`;
+const List = styled.ul`
+  margin: 0;
+  padding: var(--spacing-06) 0 0;
+`;
+const AnchorListItem = styled.li`
+  padding-top: var(--spacing-01);
+  list-style: none;
+`;
+const ListItem = styled.li`
+  padding-top: var(--spacing-01);
+  list-style-position: inside;
+  padding-left: 1.28571429em;
+  text-indent: -1.28571429em;
+`;
+const Headline1 = styled.h1`
+  margin: 0;
+`;
+const Headline2 = styled.h2`
+  padding-top: var(--spacing-09);
+  margin: 0;
+`;
+const Paragraph = styled.p`
+  padding-top: var(--spacing-06);
+  margin: 0;
+`;
+
```

Comment:
Do we not want to move these to a general styling component for these content pages? Assuming that they will be used across all the subpages. I did this using your styled components.

Reply:
Yes this is only good if we use all those elements across all subpages!